src/js/solo/top-listens/TopListeningData.jsx

```diff
@@ -141,20 +143,23 @@ class TopListeningData extends Component {
       <div className="track-data-table-container top-listens-table-container">
         <div className="track-data-table-header-container">
           <h1 className="track-data-table-header">Your Top Hits</h1>
-          <Paper square className="track-data-table-tab-panel">
-            <Tabs
+          <FormControl
+            variant="outlined"
+            className="track-data-table-tab-panel"
+          >
+            <InputLabel id="demo-simple-select-outlined-label">
+              Time Range
+            </InputLabel>
+            <Select
               value={this.state.selectedTerm}
               onChange={this.updateTopListeningDataTerm}
-              variant="fullWidth"
-              indicatorColor="primary"
-              textColor="primary"
-              aria-label="icon label tabs example"
+              label="Time Range"
             >
-              <Tab label="Short Term" />
-              <Tab label="Medium Term" />
-              <Tab label="Long Term" />
-            </Tabs>
-          </Paper>
+              <MenuItem value={0}>Recent</MenuItem>
+              <MenuItem value={1}>Recurring</MenuItem>
```

Comment:
From a user experience perspective i'm not sure it's super intuitive to understand what these mean, we can brainstorm some ideas though

Reply:
I agree, these are certainly placeholders for now